### ISSUE TITLE
notes from initial testing

### ISSUE THREAD

**Issue by moloko** [fabiobeoni/adapt-authoring-standalone#1]
You might mention that it's quite a large zip file to extract - even on my (fairly powerful) PC it took nearly 10 minutes. Equally the extracted files are 1.5GB which I think is large enough for it to be polite to warn people it will take a while and take up a fair bit of disk space!

On that note, is there anything you can do to reduce the size? I believe that typically an AT install is just 500MB or so. The authoring tool folks might have some suggestions.

I don't think you can stress enough that it needs to be extracted to `c:\adapt` - I missed this so others will too!

It would also be useful to note that a user will have to allow it via any firewall software they are running. The Windows Firewall dialogue popped up for me which I was of course expecting but a non-technical user might get a bit 'freaked out' by this so some guidance would be good. Also, for me, I gave it firewall permissions, it launched but then didn't connect. Just as I was about to start trying to figure out why, _another_ Windows Firewall warning popped up. After accepting that, it launched fine and has continued to do so.

Also might be worth pointing out that it will be necessary to allow popups for localhost so that the preview will work.

Stopping it seems to be a bit funny... it always take a while (so seems like it's not working) and sometimes fails with an error.

Overall I think this is really great idea for people wanting to have a quick try of the Authoring Tool - whether non-technical or not!

Finally just to say that I think Adapt AT v0.4 should be released quite soon so it might be worth holding off on some of this until that is ready.

**Comment by fabiobeoni** [comment]
Hi @moloko,
thanks a-lot for your comments... I'm going to work on some of the issues you reported, for others I'm not sure I can do something about.

1. Package size: unfortunately mostly depends on the AT it self, there are MongoDB/NodeJS/Adapt Authoring with all required NPM modules.
2. Unpack under **C:\adapt**: your are right... me too I don't like it so much. I actually tried to make the all package totally "portable", but then I found that during the setup of the environment (the standard Adapt Authoring install) the installation directory path is stored in a large amount of files, with-in the Adapt Authoring tool. To make the all package portable, I should replace all those references, and I'm a bit scared of doing it. That's why I decided to accept the compromise of forcing the final user to unpack under a specific path. Anyway I will try to put more emphasis on that, for sure. 
3. About the firewall, I will make it clear on the "Start" command.
4. Popups for localhost: thanks I didn't notice it, I will do something for sure
5. Stopping tacking a while: yes I will add some kind of message about what's going on, what's stopping etc..

Finally, I will wait for Adapt Authoring 0.4 before posting this project to the Adapt forum... And after those small changes, I will get in contact with the Authoring guys too.

Thanks again, your feedback is really welcome.
Ciao, Fabio.
